Assets/Spawner/Spawner.cs

```diff
@@ -773,7 +773,7 @@ public override bool AddWave(int index, Wave wave) {
 
     public override void GetSpawnTimes(ref EnemySpawnTimes spawnTimes, ref float projectedStartTime) {
       if (Finished) return;
-      projectedStartTime += delay;
+      projectedStartTime += WaveStartTime == 0.0f? delay : delay - (Time.time - WaveStartTime);
```

Comment:
dialogue box is indeterminate like waituntildeadwave.

Reply:
Done.